### DISCUSSION TITLE
Error Tunnel Cloudflare loadin NC AiO

### DISCUSSION THREAD

**Discussion by barcelona92** [nextcloud/all-in-one#1079] [Questions] [answered]
Hi,
I have a problem, when load a NC in the home not appear login (Firefox) i Chrome is possible access login, but not appear menu, not is possible access to the settings etc..
![erro-nc-cloudflare](https://user-images.githubusercontent.com/33433738/187150656-47bc054d-0318-48cf-90e5-34982789009a.png)

In the tunnel are config access to: http://192.168.20.21:11000

What happend?

**Comment by szaimen** [comment, marked as answer]
Sorry, no idea. Probably some cloudflare settings are too restrictive?

**Comment by stayathomedad** [comment]
what does your config.yml file look like? also which docker command did you use to spin up the containers?
